Makefile

```diff
@@ -36,6 +36,9 @@ modsync: fmtcheck
 	go mod tidy && \
 	go mod vendor
 
+examples:
```

Comment:
hook in the parent `Makefile` to run the child `examples/Makefile` default target